src/main/java/tc/oc/pgm/api/setting/SettingKey.java

```diff
@@ -31,7 +31,8 @@ public void update(MatchPlayer player) {
     }
   }, // Changes if observers are visible
   SOUNDS("sounds", SOUNDS_ON, SOUNDS_OFF), // Changes if sounds are played
-  ;
+  VOTE_BOOK("vote-book", VOTES_ON, VOTES_OFF) // Changes if the vote book is shown on cycle
+;
```

Comment:
Should probably leave this line unmodified

Reply:
Auto-formatter did this, checks won't pass unless it stays (I think)

Reply:
That's because you forgot about `,` at the end of the line.